google/cloud/bigquery/client.py

```diff
@@ -2562,6 +2576,13 @@ def load_table_from_dataframe(
         if job_config.source_format is None:
             # default value
             job_config.source_format = job.SourceFormat.PARQUET
+
+        if job_config.parquet_options is None:
```

Comment:
Since CSV is also supported, should we check `source_format` here?

Reply:
Yup, you're right. Probably would be fine. But also best not to be going around editing the JobConfig if not necessary.

Reply:
Based on the coverage failure, I think we need a unit test where `parquet_options` is manually set. Perhaps we can have a test where `parquet_options.enable_list_inference = False` and we verify that we don't override that?

Reply:
Done.